src/api.ts

```diff
@@ -1,8 +1,9 @@
-import axios from 'axios';
+import fetch from './fetch';
```

Comment:
Is there any particular reason why reimplementing `fetch` and not using the native [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/fetch) method?

Reply:
because native fetch not is in node LTS version. (only 17.5^)